core/store/src/trie/trie_storage.rs

```diff
@@ -171,6 +171,7 @@ impl TrieStorage for TrieCachingStorage {
         if let Some(val) = guard.cache_get(hash) {
             Ok(val.clone())
         } else {
+            tracing::debug!(target: "runtime", "trie cache miss for hash {}", hash);
```

Comment:
This might lead to too much logging. A prometheus counter may be better

Reply:
I agree here